api-test/integration-tests/src/test/java/com/microsoft/semantickernel/tests/connectors/memory/jdbc/JDBCVectorStoreRecordCollectionTest.java

```diff
@@ -0,0 +1,253 @@
+package com.microsoft.semantickernel.tests.connectors.memory.jdbc;
+
+import com.microsoft.semantickernel.connectors.data.jdbc.JDBCVectorStoreRecordCollection;
+import com.microsoft.semantickernel.connectors.data.jdbc.JDBCVectorStoreRecordCollectionOptions;
+import com.microsoft.semantickernel.connectors.data.jdbc.MySQLVectorStoreQueryProvider;
+import com.microsoft.semantickernel.data.recordoptions.GetRecordOptions;
+import com.microsoft.semantickernel.tests.connectors.memory.Hotel;
+import com.mysql.cj.jdbc.MysqlDataSource;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import javax.annotation.Nonnull;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@Testcontainers
+public class JDBCVectorStoreRecordCollectionTest {
+    @Container
+    private static final MySQLContainer<?> CONTAINER = new MySQLContainer<>("mysql:5.7.34");
```

Comment:
Maybe use [maven resource filtering](https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html) to automagically supply the mysql version.

Reply:
Setting it to latest